armi/physics/fuelCycle/tests/test_fuelHandlers.py

```diff
@@ -736,6 +736,127 @@ def test_swapFluxParamDifferentLengths(self):
         for b in a2.getBlocks():
             self.assertEqual(b.p.flux, 50000000000.0)
 
+    def test_transferStationaryBlocks(self):
```

Comment:
I love these tests!

Can we also get a test or two for `dischargeSwap()`, that shows this feature working.

Thanks!